docs/source/pipelines.rst

```diff
@@ -183,57 +188,51 @@ The general workflow for using MLflow Pipelines is as follows:
           # Inspect the resulting model performance evaluations
           mlflow pipelines inspect --step evaluate --profile local
 
-    .. note::
-      Each :ref:`Pipeline Template <pipeline-templates>` also includes a |Databricks Notebook|
-      and a |Jupyter Notebook| for running the pipeline and inspecting its results.
 
-      Example pipeline run from the |Databricks Notebook| included in the
-      |MLflow Pipelines Regression Template|:
+    .. figure:: _static/images/pipelines_evaluate_step_card.png
+      :width: 60%
 
-      .. figure:: _static/images/pipelines_databricks_notebook_ui.png
-        :scale: 25
+      An example step card produced by running the **evaluate** step of the
+      :ref:`MLflow Regression Pipeline <mlflow-regression-pipeline>`. The step card results
+      indicate that the trained model passed all performance validations and is ready for
```

Comment:
Not a comment, but github thought `all` was important and I thought it looked funny
<img width="338" alt="image" src="https://user-images.githubusercontent.com/51172624/197676902-f4791249-e76a-4e9b-9090-d11986868929.png">